spec/support/system_description_factory.rb

```diff
@@ -782,6 +867,27 @@ def build_description(name, store, options)
       ]
     }
   EOF
+
+  EXAMPLE_SCOPES["users_with_passwords_centos"] = <<-EOF.chomp
```

Comment:
same as for the groups, I would use the already existing example user scope `users_with_passwords`.